indico/modules/events/timetable/client/js/WeekViewToolbar.tsx

```diff
@@ -27,7 +24,7 @@ export default function Toolbar({
   const numDays = useSelector(selectors.getEventNumDays);
   const offset = useSelector(selectors.getNavbarOffset);
 
-  const getDateFromIdx = idx => moment(eventStart.getTime() + idx * 24 * 60 * 60 * 1000);
+  const getDateFromIdx = idx => moment(eventStart).add(idx, 'days');
```

Comment:
Just wondering, do we want to use moment in the new timetable code? Does it provide much usefulness compared to the native datetime functions nowadays?

Reply:
That's definitely up for debate, the current date handling is still kind of messy so it will need cleanup anyway. With moment I was also a bit worried about performance but I haven't done any profiling yet.